Python/Product/Analysis/Interpreter/Ast/AstPythonModule.cs

```diff
@@ -130,6 +135,8 @@ sealed class AstPythonModule : IPythonModule, IProjectEntry, ILocatedMember {
             var walker = new AstAnalysisWalker(interpreter, ast, this, filePath, DocumentUri, _members, true, true);
             ast.Walk(walker);
             walker.Complete();
+
+            ParseErrors = parseErrors.ToArray();
```

Comment:
`parseErrors` can be null, causing ANE on this line

Repro with test `UnassignedClassMembers` and `OsPathMembers`